cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -808,6 +813,28 @@
         }
       ]
     },
+    "CreateFSXSubstack": {
+      "Fn::Not": [
+        {
+          "Fn::Equals": [
+            {
+              "Fn::Select": [
+                "0",
+                {
+                  "Fn::Split": [
```

Comment:
minor: I think you don't need to split in a comma separated list

Reply:
The list is passed in as a string